learn/getting-started/writing-a-grpc-service-with-ballerina.md

```diff
@@ -144,52 +176,68 @@ Running executable
 
 Now you completed the server-side implementation, and it is running on port 9090. Let’s move on to the gRPC client-side implementation.
 
-### Developing a gRPC client
+## Implementing the gRPC client
 
 Similar to the service, the client application also starts with creating a new Ballerina project. Once created, you can generate the client code and update the code to call the remote methods exposed by the service.
 
-#### Creating a Ballerina project
+### Creating the client project
 
-Execute the following command to create a new Ballerina project in the directory that you want to create the project.
+In a new tab of the terminal, navigate to the `grpc` directory, and execute the command below to create the Ballerina project for the gRPC client implementation:
+
+> **Note:** For more information on Ballerina projects, see [Organizing Ballerina code](/learn/organizing-ballerina-code/).
 
 ```bash
 $ bal new greeter_client
 ```
 
-> **Note:** For more information on creating Ballerina packages, see [Getting Started with Ballerina](/learn/getting-started-with-ballerina).
+You view the output below.
+
+```bash
+Created new package 'greeter_client' at greeter_service.
+```
+
+This creates a directory named `greeter_client` with the files below.
 
-Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
+```bash
+.
+├── Ballerina.toml
+└── main.bal
+```
 
-#### Generating the source code
+>**Tip:** Remove the automatically-created `main.bal` file as you are not going to use it in this guide.
 
-Execute the following command inside the same directory in which you created the Ballerina project to generate the source code related to the service definition.
+### Generating the source code of the client
+
+In the terminal, from inside the same `grpc` directory, execute the command below to generate the source code related to the client definition.
 
 ```bash
 $ bal grpc --mode client --input helloworld.proto --output greeter_client/
 ```
 
-Once successfully executed, you could be able to view the below output.
-```
+Once successfully executed, you will see the output below.
+
+```bash
+Continuing with the existing protoc executor file at /var/folders/6p/8h4k83hj0r98n1s6mc3jy0480000gn/T/protoc-3.9.1-osx-x86_64.exe
```

Comment:
This output line varies depending on whether the executor is already downloaded or not. Here it is already downloaded. Shall we remove this line?

Reply:
Removed in https://github.com/ballerina-platform/ballerina-dev-website/pull/4069/commits/ebff86fbdafea4347d3b8a88f47610696e5787d9.